octoprint_mrbeam/templates/laserhead_changed.jinja2

```diff
@@ -9,6 +9,7 @@
             <h4> {{ _("Your new Mr Beam dreamcut [S] Laserhead has been successfully installed.") }}</h4>
             <p> {{ _("With this Laserhead you can work now even faster.") }}</p>
             <p> {{ _("This means that we also update the default material settings to the new laserhead as long it is installed to provide you the best possible support. However, we do not update your custom materials, this will need your personal testing and finetuning.") }}</p>
+            <p> {{ _("Read about what changes you can expect with a new and improved Laserhead Mr Beam II dreamcut [S].") }} <a target="_blank" href="{{ _("https://support.mr-beam.org/en/support/solutions/articles/43000640531") }}"> » {{ _("Read more") }} </a></p>
```

Comment:
1. 
> Read about changes

Or 

> Read about the changes

2. Funky name "Laserhead Mr Beam II dreamcut [S]"

Reply:
perhaps "Laserhead for MrBeam II dreamcut [S]" is less of a mouthful.

Reply:
-> with _the_ new and improved

Reply:
fixed

Reply:
But this laserhead is not olny for the dreamcut [S]